web/src/main/java/nl/tehdreamteam/se42/web/controller/UserController.java

```diff
@@ -0,0 +1,21 @@
+package nl.tehdreamteam.se42.web.controller;
+
+import nl.tehdreamteam.se42.domain.User;
+import nl.tehdreamteam.se42.web.Service;
+
+/**
+ * Defines functions that specific {@link Service Services} can use to serve content.
+ */
+public interface UserController {
+
+    /**
+     * Logs a {@link User} in, given that the specified {@code username} and {@code password} are correct, and returns
+     * a {@code token} that temporarily identifies the logged-in {@code User}.
+     *
+     * @param username The {@code username} of the {@code User} to login.
+     * @param password The {@code password} of the {@code User} to login.
+     * @return A {@code token} that temporarily identifies the logged-in {@code User}.
+     */
+    String login(String username, String password); // TODO, should not be a String, probably a dedicated Token class
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1135)